src/main/java/com/compstore/service/impl/PCServiceImpl.java

```diff
@@ -105,6 +108,7 @@ public void deletePCById(UUID pcId) {
         log.info("Requested deleting a PC with id: {}", pcId);
         Optional<PCEntity> pcById = pcRepository.findById(pcId);
         if (pcById.isPresent()) {
+            pcValidator.validateDelete(pcById.get());
```

Comment:
Zauważyłem błąd 500 na usuwaniu peceta, gdy był wykorzystywany w zamówieniu. Musiałem zrobić walidację sprawdzenia użycia `pc`. Później będzie także do dorobienia w innych encjach dziedziczących po `product`, lecz to później